index.js

```diff
@@ -79,7 +79,8 @@ if (process.env.TRAVIS) {
   event = process.env.GITHUB_EVENT_NAME
   commit_message = ''
   pull_request_number = ''
-  branch = process.env.GITHUB_REF
+  // GITHUB_HEAD_REF for pull requests. For commits, GITHUB_REF is of the form refs/heads/master, for example
+  branch = process.env.GITHUB_HEAD_REF || process.env.GITHUB_REF && process.env.GITHUB_REF.split('/')[2]
```

Comment:
Fwiw, `process.env.GITHUB_REF.split('/')[2]` will almost certainly break various things.

By default, @dependabot uses `/` a lot in its branch names:
> Dependabot generates a branch for each pull request. Each branch name includes `dependabot`, and the package manager and dependency that are updated. By default, these parts are separated by a `/` symbol, for example: `dependabot/npm_and_yarn/next_js/acorn-6.4.1`.


https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#pull-request-branch-nameseparator

(And random people can use it as well, especially people in large organizations.)

I'm only mentioning this because my GitHub workflow broke when someone used `/` and I didn't have the right branch flags...